src/schemas/genes.json5

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
How is the key ID defined?  We may want to document that somehow.  

Specifically for the genome and gene, I would like to have some discussion about the key.  Are these going to be explicitly linked to the WS object?  If so, then it would be good if the key mapped to the object.  e.g.

id: WS:1/2/3 for a genome or other node that mapped directly to an object.

for feature or other sub-objects we should have some convention.  e.g. 

id: WS:1/2/3:feature/123

This roughly mirrors what is done in search.

The other question is do we have other genome-like and gene-like nodes in the graph that don't map directly to a WS object.

Reply:
- good point, we should include `_key` in all schemas. FYI, JSON Schema can include metadata like "description", "default", and "examples" for every field.
- it might also be worth having a separate "workspace_ref" field for documents in the db that map to a workspace object (eg. a genome). I'm not familiar enough with the KBase workspace to know all the stuff there to know how things might map